index.rkt

```diff
@@ -66,6 +66,7 @@
   }}}
   @div[class: "row"]{
     @research-highlights[
+      @to-appear[@kas-scheme-2018]
```

Comment:
Please move this down to line 80 --- after the SPLASH papers --- so everything's in chronological order.

Also `@to-appear` can be `@appeared` :)

Reply:
done.